### PR TITLE
Prevent alternative hide methods based on webaim standard

### PR DESCRIPTION
**Pull request by NickIannelli**

### CODE REVIEW

**Comment by carloscheddar** [comment]
@nickShopAt Good stuff but the `width` and `height` properties are causing display issues for some subs like [/r/politics](https://www.reddit.com/r/politics/). Maybe `min-width` and `min-height` could be used instead.

**Comment by NickIannelli** [comment]
@carloscheddar Good call, will update.

File: styles.css [hunk 1]
```diff
@@ -1,4 +1,15 @@
-body .content .midcol > .arrow {
+body .content .midcol > .arrow,
+body .content .midcol > .arrow:hover,
+body .content .midcol > .arrow:active {
   display: block !important;
   visibility: visible !important;
+  text-indent: 0!important;
+  clip: initial!important;
```
Comment (carloscheddar) [inline styles.css]: Could you add a space between the value and `!important` this way it matches the existing properties.

Comment (NickIannelli) [reply styles.css]: Done. Apologies - my css-lint config is for the opposite.

**Comment by carloscheddar** [comment]
Nice work! I'm going to merge this and release a new version soon.

**Pull request merged**